datavault/__init__.py

```diff
@@ -216,26 +217,30 @@ def exclude(entries, tag, tags):
                 tag = tag[1:]
             else:
                 filter = include
-            #print filter.__name__ + ':', tag
-            #print 'before:', dirs, datasets
             dirs = filter(dirs, tag, self.session_tags)
             datasets = filter(datasets, tag, self.dataset_tags)
-            #print 'after:', dirs, datasets
         return dirs, datasets
 
     def listDatasets(self):
         """Get a list of dataset names in this directory."""
         files = os.listdir(self.dir)
-        files.sort()
-        return [filename_decode(s[:-4]) for s in files if s.endswith('.csv') or s.endswith('.bin')]
-
-    def newDataset(self, title, independents, dependents):
+        filenames = []
+        for s in files:
+            base, _, ext = s.rpartition('.')
+            if ext in ['csv', 'hdf5']:
```

Comment:
`.bin` was an experiment that happened on a branch that this branch was derived from. We decided to go for hdf5 instead. I think this must have made it into master by mistake when we merged some other features from that branch. We've never actually used a version of the data vault that wrote `.bin` files.